use std::process::exit;

// success denoted by non-zero return
const DEFAULT_ERROR_CODE: i32 = 1;

fn main() {
    exit(DEFAULT_ERROR_CODE);
}